base/essentials.jl

```diff
@@ -163,6 +163,7 @@ true
 """
 function convert end
 
+convert(::Type{Union{}}, x) = throw(MethodError(convert, (Union{}, x)))
```

Comment:
Is this here to give a predictable ambiguity error instead of an ambiguity error involving two random methods? I suppose that's fine, just wondering if there is any other reason.

Reply:
As you say, "predictability" :)